extensions/playerlist/in-div.ejs

```diff
@@ -0,0 +1,13 @@
+<style type="text/css">
+  #playerlist span {
+    margin: 5px;
+  }
+</style>
+
+<div id="playerlist-overlay">
+  <h2>Player List</h2>
+  <input type="text" id="searchPlayer">
```

Comment:
Optional: Consider adding an onchange event callback so users will not have to wait 1 second for the update.

Reply:
Added to TODO